packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -140,6 +131,15 @@ export default class ButtonCore extends React.Component<Props> {
 
         const contents = (
             <React.Fragment>
+                {icon && (
+                    <Icon
+                        size={iconSize}
+                        color="currentColor"
+                        icon={icon}
+                        style={sharedStyles.icon}
+                        aria-disabled="true"
```

Comment:
**suggestion:** probably using `aria-hidden` here would be better. That way the icon is not announced by screen readers. Did you add `aria-disabled` for a different purpose?

```suggestion
                        aria-hidden="true"
```

Reply:
my bad! i think i mixed up the aria labels. i'll update that